modules/apps/frontend-token/frontend-token-definition-test/src/testIntegration/java/com/liferay/frontend/token/definition/test/FrontendTokenDefinitionRegistryTest.java

```diff
@@ -90,37 +90,48 @@ public void tearDown() throws PortalException {
 
 	@Test
 	public void testGetFrontendTokenDefinitions() {
+		List<FrontendTokenDefinition> frontendTokenDefinitionsList =
```

Comment:
In [this commit](https://github.com/liferay-platform-experience/liferay-portal/pull/740/commits/38a1cb541b2b2448b425a5ceda3947ec83bdfc4b#diff-8b40ded19821a2282c5245e447b66bd5de1f8eeb0093075ebbffbec86abe17c0R80), Brian renamed the list to `frontendTokenDefinitionsList`, but that's because there is a `frontendTokenDefinitions` (which is a map) already. Other than that, you call it `frontendTokenDefinitions`